src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -64,79 +51,46 @@ public long RawContentLength
         /// </value>
         public string RawContent { get; protected set; }
 
-        #endregion Properties
-
-        #region Methods
-
         /// <summary>
-        /// Reads the response content from the web response.
+        /// Gets the length (in bytes) of <see cref="RawContentStream"/>.
         /// </summary>
-        private void InitializeContent()
-        {
-            this.Content = this.RawContentStream.ToArray();
-        }
-
-        private static bool IsPrintable(char c)
+        public long RawContentLength
         {
-            return (char.IsLetterOrDigit(c) || char.IsPunctuation(c) || char.IsSeparator(c) || char.IsSymbol(c) || char.IsWhiteSpace(c));
+            get { return (RawContentStream == null ? -1 : RawContentStream.Length); }
```

Comment:
Here and below we could remove extra braces and simplify code:
```suggestion
            get => RawContentStream is null ? -1 : RawContentStream.Length;
```